tasks/util/constants.js

```diff
@@ -13,13 +12,27 @@ var pathToTopojsonSrc = path.join(
     path.dirname(require.resolve('sane-topojson')), 'dist/'
 );
 
+var partialBundleNames = [
```

Comment:
I guess it's not that weird to modify the constants file to get it to compile your own bundle. I'm fine with this approach + documenting how to bundle your own. The market for people who want to bundle their own but who don't want to read the instructions and just edit this line might be vanishingly small.